justfile

```diff
@@ -37,7 +37,7 @@ hugepages_1g := "8"
 hugepages_2m := "1024"
 _just_debuggable_ := if debug == "true" { "set -x" } else { "" }
 target := "x86_64-unknown-linux-gnu"
-profile := "dev"
+profile := "debug"
```

Comment:
This is a fine way to change the default profile, but I think we need a little more to resolve the root issue here.

We will also need the changes currently part of #201 as well as a test in CI to confirm that our build artifacts are statically linked